cognite/src/dto/iam/session.rs

```diff
@@ -0,0 +1,77 @@
+use std::fmt::Display;
+
+use serde::{Deserialize, Serialize};
+
+use crate::{to_query, AsParams, SetCursor};
+
+#[derive(Serialize, Deserialize)]
```

Comment:
```suggestion
#[derive(Serialize, Deserialize)]
#[serde(rename_all = "camelCase")]
```
to avoid the individual `#[serde(rename = ...)]`?

Reply:
I think it's technically `snake_case` here, but sure, that's probably better.